subgraph/bin/networks.js

```diff
@@ -46,14 +48,53 @@ const generateNetworksFile = async () => {
   console.log(`Networks file saved at: ${networkFilePath}`)
 }
 
+function setupFolderConfig() {
+  // make sure we clean up
+  if (fs.pathExistsSync(configFolderPath)) {
+    fs.rmSync(configFolderPath, { recursive: true, force: true })
+  }
+  fs.mkdirSync(configFolderPath)
+}
+
+function createNetworkAbi(network, chainName) {
```

Comment:
any specific reason to chose the name `createNetworkAbi` ? The files creating seems to be config files not ABIs

Reply:
Will adjust, apologies.

Reply:
No worries at all !